README.md

```diff
@@ -1 +1,72 @@
-# My-portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My-Portfolio
+
+> One paragraph statement about the project.
+
+![screenshot](./img/mobile-skeleton.png
+)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Html
+- Css
+
+## Live Demo
+
+[Live Demo Link](https://chukwuebukavictor.github.io/My-portfolio/)
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+    - VS Code
+    -Github
+
+### Setup
+   - git clone your repo
+   - cd into the folder
+   - open index.html with live server
+
+### Install
+    VS Code
+
+### Usage
+    This is a simple project to dipslay my portfolio
+
+
+## Authors
+
+👤 **Author1**
```

Comment:
Instead of **Author1** I recommend you add your name.

- [ ] Add your name in the Authors section.